eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/steps/processors/VariantProcessor.java

```diff
@@ -0,0 +1,16 @@
+package uk.ac.ebi.eva.accession.pipeline.steps.processors;
+
+
+import org.springframework.batch.item.ItemProcessor;
+
+import uk.ac.ebi.eva.accession.core.ISubmittedVariant;
+import uk.ac.ebi.eva.commons.core.models.pipeline.Variant;
+
+public class VariantProcessor implements ItemProcessor<Variant, ISubmittedVariant> {
+
+    @Override
+    public ISubmittedVariant process(Variant variant) throws Exception {
```

Comment:
shouldn't this take `IVariant`?

Reply:
and what concrete object should we return? VariantDTO is in the WS package, and VariantEntity doesn't feel right

Reply:
agreed to implement `ItemProcessor<IVariant, VariantEntity>`